docs/UPGRADING.md

```diff
@@ -1,6 +1,6 @@
 # Upgrading
 
-The `connectors-python` project is intentionally de-coupled from the Elasticsearch release cycle.
+The `connectors` project is intentionally de-coupled from the Elasticsearch release cycle.
```

Comment:
```suggestion
The Elastic connectors project is intentionally de-coupled from the Elasticsearch release cycle.
```

or 
```
Connectors are intentionally de-coupled...
```

WDYT? Feels like "connectors project" just sounds odd.

Reply:
Fair, sounds odd. I'll update it and remove `projects`.